invenio_notifications/backends/backend.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```

Comment:
minor: we can rename this file to `base.py` (to follow similar patterns as in other Invenio modules)